js/views/attachment_view.js

```diff
@@ -25,15 +27,16 @@
         this.trigger('update');
       },
       render: function() {
-        this.$el.attr('src', this.dataUrl);
+        this.$el.attr('src', this.blobUrl);
         return this;
       }
   });
 
   var MediaView = Backbone.View.extend({
-      initialize: function(dataUrl, contentType) {
+      // TODO: Disable JSHint to allow for `{contentType}={}` syntax:
```

Comment:
Is there anything blocking us from dealing with this TODO now?

Reply:
Nope, removed it: 8cf84cf473668b11e9da22ef4e5543a2e765f407

Reply:
Update: Opted the whole file into ESLint: efa35b647ba78086785511f69089a9bfcdebe680